src/nodes.coffee

```diff
@@ -1104,47 +1115,77 @@ exports.Obj = class Obj extends Base
 
   children: ['properties']
 
+  isAssignable: ->
+    @lhs = true
```

Comment:
Wait a minute . . . is `@lhs` hacky because it’s getting _set_ via the `isAssignable` call? And you’re relying on the fact that `isAssignable` is only called when `Obj` is part of an `Assign` and therefore the left hand side of a destructuring assignment?

If that’s the case, then yeah, this is really hacky. `isAssignable` is not the name for a method that should be mutating its class. We should find a better way.